README.md

```diff
@@ -1,7 +1,12 @@
 # Cluster Ingress Operator
 
-The Cluster Ingress Operator manages highly available network ingress for [OpenShift](https://openshift.io/). Simplicity is a design goal: minimal configuration should be required to materialize a highly available ingress solution in both cloud and bare metal environments.
+Cluster Ingress Operator deploys and manages [HAProxy](http://www.haproxy.org) to provide highly available network ingress in [OpenShift](https://openshift.io/), enabling both OpenShift [Routes](https://docs.okd.io/latest/rest_api/apis-route.openshift.io/v1.Route.html) and Kubernetes [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).
 
-## Development
+The operator tries to be useful out of the box by creating a working default deployment based on the cluster's configuration.
 
-See [HACKING](HACKING.md) for development topics.
+* On cloud platforms, ingress is exposed externally using Kubernetes [LoadBalancer Services](https://kubernetes.io/docs/concepts/services-networking/#loadbalancer).
+* On Amazon AWS, wildcard DNS for the ingress domain is automatically managed.
```

Comment:
How about, "the operator publishes a wildcard DNS record based on the [cluster ingress domain](https://github.com/openshift/api/blob/master/config/v1/types_ingress.go) and pointing to the service's load-balancer"?

Reply:
Fixed anyway in advance of the PR since merge is imminent